src/server/lock_manager/deadlock.rs

```diff
@@ -1030,6 +1545,9 @@ where
             } => {
                 self.handle_detect(tp, txn_ts, wait_info, diag_ctx);
             }
+            Task::ReplaceLocksByKeys { items } => {
+                self.handle_replace_locks_by_keys(items);
+            }
```

Comment:
We've observed that the cpu util of deadlock-detector thread is extremely high from the issues we encountered previously. Should we also add some metrics to observe the cpu time consumed by each kind of task handling?

Reply:
Sounds good. How to calculate CPU time per task?

Reply:
I think we can simply record the duration of `Runnable::run` by task type (maybe also distinguish local/remote handling). The thread util (by task) should be equal to `sum by (type) (rate(xxx_duration))` which consist of cpu time and io time. The io time should be negligible for deadlock-detector thread, otherwise it would become the bottleneck firstly (and we can observe that by comparing thread util and cpu util).